pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -260,6 +264,22 @@ func validateNewMacAddresses(vmClone *clone.VirtualMachineClone) []metav1.Status
 	return causes
 }
 
+func validatePatches(vmClone *clone.VirtualMachineClone) []metav1.StatusCause {
```

Comment:
There is some value in this validation, but it's still pretty weak. Valid JSON doesn't have to be a valid JSON patch. Can you follow up with an extended test?